lib/tf_lock/tf_lock_acquire.py

```diff
@@ -43,6 +46,31 @@ def tf_lock_acquire(root_module: Path) -> ExitCode:
                 )
                 return 0
 
+            else:
+                p = Parse(lock_user)
+                username = p.before.first("@")
+                # a pr holds the lock.
+                if "github" in str(lock_user):
+                    pr_number = p.after.first("@").before.first(".")
+                    repo_name = p.after.first(".").before.last(".", ".", ".")
+                    org_name = p.after.first(repo_name, ".").before.last(
+                        ".github"
+                    )
+
+                    if is_pr_closed(org_name, repo_name, pr_number):
+                        sh.info("forcing unlock on a closed PR.")
+                        force_unlock(root_module)
```

Comment:
i think we need to `continue` after force_unlock(). otherwise we will return the error even if unlock succeeds